examples/custom/contrib/FHEM/HMConfig_AskSinPPCustom.pm

```diff
@@ -72,6 +72,15 @@ $HMConfig::culHmRegDefine{"longActNum"}                   = {a=>0xA5,s=>1.0,l=>3
 $HMConfig::culHmRegDefine{"longActIntens"}                = {a=>0xAB,s=>1.0,l=>3,min=>0,max=>255,c=>'',p=>'y',f=>'',u=>'',d=>1,t=>"Volume or Led Brightness"};
 # Volume0=0, Volume10=20, Volume20=40, Volume40=60, .... Volume100=200
 
+# HB-UNI-Sensor-THPD-BME280
+$HMConfig::culHmRegDefine{"deviceLedMode"} = {a=>5.6,s=>0.2,l=>0,min=>0,max=>1,c=>'',p=>'n',f=>'',u=>'',d=>1,t=>"LED mode"};
+$HMConfig::culHmRegDefine{"lowBatLimit"} = {a=>18,s=>1,l=>0,min=>0.9,max=>5.0,c=>'',p=>'n',f=>10,u=>'V',d=>1,t=>"low battery limit"};
+$HMConfig::culHmRegDefine{"transmitDevTryMax"} = {a=>20,s=>1,l=>0,min=>1,max=>10,c=>'',p=>'n',f=>'',u=>'',d=>1,t=>"transmitDevTryMax"};
```

Comment:
can we reuse this https://github.com/mhop/fhem-mirror/blob/41fe9bbf317f54bccfc397e26224c711e42525a8/fhem/FHEM/HMConfig.pm#L526